docs/apache-airflow-providers-apache-kafka/connections/kafka.rst

```diff
@@ -0,0 +1,38 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection: kafka
+
+Apache Kafka Connection
+========================
+
+The Apache Kafka connection type configures a connection to Apache Kafka via the ``confluent-kafka`` Python package.
+
+Default Connection IDs
+----------------------
+
+Kafka hooks and operators use ``kafka_default`` by default.
```

Comment:
Are we missing definition for `kafka_default` in
https://github.com/apache/airflow/blob/ffa2d479e93e452faca1eb1b25c49098f2a98d63/airflow/utils/db.py#L113

or is it intentional not to have it?

Reply:
Good one.That reminds me that is about the last thing where we will have "core" referring to particular "providers" (once we move executors to providers). One more last step to truly decouple providers from the core.

And yeah. I think we should have a default connection here.

Reply:
Just an oversite - updated.